examples/CIB/ex1/main.cpp

```diff
@@ -379,8 +430,53 @@ main(int argc, char* argv[])
                 navier_stokes_integrator->setStokesSolverNeedsInit();
             }
 
+            int coarsest_ln = 0;
+            int finest_ln = patch_hierarchy->getFinestLevelNumber();
+
+            // Update the location of the box for time n + 1
+            hydro_force->updateStructureDomain(0, dt, Eigen::Vector3d::Zero(), patch_hierarchy);
```

Comment:
Suggest switching the calling sequence to put the structure ID last, and giving a default ID of 0.